usage/rest-server/pom.xml

```diff
@@ -241,9 +241,46 @@
                 <!-- Required to set values in build-metadata.properties -->
                 <filtering>true</filtering>
             </resource>
+            <resource>
+                <directory>${basedir}/src/main/webapp</directory>
+            </resource>
         </resources>
         <plugins>
 
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <!-- configure plugin to generate MANIFEST.MF
+                     adapted from http://blog.knowhowlab.org/2010/06/osgi-tutorial-from-project-structure-to.html -->
+                <executions>
+                    <execution>
+                        <id>bundle-manifest</id>
+                        <phase>process-classes</phase>
+                        <goals>
+                            <goal>manifest</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <supportedProjectTypes>
+                        <supportedProjectType>jar</supportedProjectType>
+                    </supportedProjectTypes>
+                    <instructions>
+                        <Import-Package>
+                            com.sun.jersey.spi.container.servlet;version="[1.18.1,2.0)",
+                            *
+                        </Import-Package>
+                        <Export-Package>org.apache.brooklyn.*</Export-Package>
+                        <Implementation-SHA-1>${buildNumber}</Implementation-SHA-1>
+                        <Implementation-Branch>${scmBranch}</Implementation-Branch>
```

Comment:
Yes, they are indeed. However, I plan to revisit each pom regarding maven-bundle-plugin, so I slacked a bit.